docs/source/content/overview/linear_SVM.rst

```diff
@@ -2,67 +2,157 @@
+The data that is closest to the line are what determine the optimal line. These data points are called 
+**support vectors**. They are shown as the filled in squares and circles above. The distance from these vectors to the
+hyperplane is called the **margin**. In general, the further those points are from the hyperplane, the greater the 
+probability of correctly classifying the data. There is a lot of complex math that goes into finding the support vectors
+and maximizing the margin. We won't go into that; we just want to get the basic idea behind SVMs. 
 
 Ignore Outliers
 ---------------
 
-Sometimes data classes will have **outliers**, (data that is separated from it's class). Support Vector Machines will ignore these outliers. This is shown in the figure below. 
+Sometimes data classes will have **outliers**. This is data points that are clearly separated from the rest of its class.
+Support Vector Machines will ignore these outliers. This is shown in the figure below. 
 
-.. figure:: https://github.com/machinelearningmindset/machine-learning-for-everybody/blob/master/docs/source/content/overview/_img/SVM_Outliers.png
-   :scale: 50 %
+
+.. figure:: _img/SVM_Outliers.png
+   :scale: 50%
    :alt: Outliers
 
    Ref:  https://www.analyticsvidhya.com/blog/2017/09/understaing-support-vector-machine-example-code/
 
-The star that is with the red circles is the outlier. So the SVM ignores the outlier and creates the best line to separate the two classes. 
+The star that is with the red circles is the outlier. So, the SVM ignores the outlier and creates the best line to separate
+the two classes. 
 
 
-Kernal SVM
+Kernel SVM
 -----------
 
-There will be data classes that can't be separated with a simple line or hyperplane. This is called **non-linearly separable data**. Here is an example of that kind of data. 
+There will be data classes that can't be separated with a simple line or hyperplane. This is called **non-linearly 
+separable data**. Here is an example of that kind of data. 
 
-.. figure:: https://github.com/machinelearningmindset/machine-learning-for-everybody/blob/master/docs/source/content/overview/_img/SVM_Kernal.png
-   :scale: 50 %
+.. figure:: _img/SVM_Kernal.png
+   :scale: 50%
    :alt: Outliers
 
    Ref:  https://www.analyticsvidhya.com/blog/2017/09/understaing-support-vector-machine-example-code/
 
 
-There's is no clear way to separate the stars from the circles. We will go over how SVMs will be able to classify non-lenearly separable data in our section about **Kernal SVM**. 
+There's is no clear way to separate the stars from the circles. SVMs will be able to classify non-linearly separable
+data by using a trick called the **kernel trick**. Basically, what goes behind the scenes, is that it takes the points
```

Comment:
Basically, the kernel trick takes the points to a higher dimension to turn non-linearly ...